config/samples/v1beta1_verticadb.yaml

```diff
@@ -15,12 +15,13 @@ apiVersion: vertica.com/v1beta1
 kind: VerticaDB
 metadata:
   name: verticadb-sample
+
```

Comment:
minor nit: can we just undo the changes in here entirely. I know semantically nothing changed, I would prefer if this file didn't change at all